twake/frontend/src/app/services/languages/locale/en.js

```diff
@@ -1852,4 +1852,5 @@ export default {
   'scenes.client.mainview.tabs.tabstemplateeditor.title_tab_creation': 'Create a new tab',
   'scenes.client.mainview.tabs.tabstemplateeditor.title_tab_edition': 'Rename $1',
   'components.connectorslistmanager.add_connectors': 'Add connectors',
+  'components.notifications.hidden_notifications_button.children': 'Hidden notifications',
```

Comment:
Such text message is not tech oriented, it should be "New messages" (or something like that)

Reply:
Replaced to `New messages`, thank you @chamerling :smiley: 
![Capture d’écran de 2020-12-15 16-55-49](https://user-images.githubusercontent.com/36481167/102246967-a32ab600-3eff-11eb-87e0-c3b007ebd859.png)